### PR TITLE
feature/identity-models

### PR DESCRIPTION
**Pull request by ahmednasserzaza**

### CODE REVIEW

**Review by NadaFeteiha — APPROVED** [review]

**Review by Shehab-m — APPROVED** [review]

File: service_identity/src/main/kotlin/org/thechance/service_identity/data/dto/UserDetailsDto.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package org.thechance.service_identity.data.dto
+
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.Serializable
+import org.bson.codecs.pojo.annotations.BsonId
+import org.litote.kmongo.Id
+
+@Serializable
+data class UserDetailsDto(
```
Comment (Asiasama710) [inline service_identity/src/main/kotlin/org/thechance/service_identity/data/dto/UserDetailsDto.kt]: I think this is should be  `UserDetailsCollection` not `dto ` these will be handled in the database and mapping it to `entity `and then `dto `that using it in endpoint

Comment (ahmednasserzaza) [reply service_identity/src/main/kotlin/org/thechance/service_identity/data/dto/UserDetailsDto.kt]: Done ✔
Thanks, Asia

File: service_identity/src/main/kotlin/org/thechance/service_identity/utils/PermissionType.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package org.thechance.service_identity.utils
+
+enum class PermissionType {
+    CLIENT,
+    ADMIN,
+    DELIVERY,
+    TAXI_DRIVER,
```
Comment (devsadeq) [inline service_identity/src/main/kotlin/org/thechance/service_identity/utils/PermissionType.kt:7]: What the difference between delivery and taxi driver?

**Review by devsadeq — APPROVED** [review]
LGTM

**Pull request merged**